src/components/Dots.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Dot from './Dot';
+import './Dots.css';
+
+const Dots = ({ index, quantity, dotClick }) => {
+  let dots = [];
+  for (let i = 0; i < quantity; i++) {
+    let isActive = i === index ? true : false;
+    dots.push(<Dot key={i} id={i} active={isActive} dotClick={dotClick} />);
```

Comment:
think `active={i===index}` should be enough

Reply:
Thanks.